addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -353,6 +353,11 @@ export class OdooEditor extends EventTarget {
         this.addDomListener(this.document, 'mousedown', this._onDoumentMousedown);
         this.addDomListener(this.document, 'mouseup', this._onDoumentMouseup);
 
+        // when you click outisde the parent iframe
+        if (window.top != this.document.defaultView) {
```

Comment:
Be careful that in javascript `!=` is different from `!==`. 99.999% of the time `!==` is what we're looking for and this case doesn't break the rule. See [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality).

```suggestion
        if (window.top !== this.document.defaultView) {
```